From 84b54ee2ff01005f0201c51f50985faf4e79edc6 Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.org>
Date: Mon, 10 Dec 2018 17:35:58 +0000
Subject: [PATCH] staging: vc-sm-cma: Remove obsolete comment and make
 function static

Removes obsolete comment about wanting to pass a function
pointer into mmal-vchiq as we now do.
As the function is passed as a function pointer, the function itself
can be static.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.org>
---
 drivers/staging/vc04_services/vc-sm-cma/vc_sm.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

--- a/drivers/staging/vc04_services/vc-sm-cma/vc_sm.c
+++ b/drivers/staging/vc04_services/vc-sm-cma/vc_sm.c
@@ -594,8 +594,7 @@ error:
 	return ret;
 }
 
-/* FIXME: Pass a function pointer to this into vc_vchi_sm.c */
-void
+static void
 vc_sm_vpu_event(struct sm_instance *instance, struct vc_sm_result_t *reply,
 		int reply_len)
 {
